common/baselines/tedlium2/hybrid/baseline_config.py

```diff
@@ -0,0 +1,71 @@
+import copy
+from sisyphus import gs, tk
+
+from i6_core.features import FilterbankJob
+
+from i6_experiments.common.setups.rasr.util import RasrSteps
+from i6_experiments.common.setups.rasr.hybrid_system import HybridSystem
+from i6_experiments.common.baselines.tedlium2.default_tools import RETURNN_RC_ROOT, RASR_BINARY_PATH
+
+from .data import get_corpus_data_inputs
+from .baseline_args import get_log_mel_feature_extraction_args
+from .nn_config.nn_args import get_nn_args
+
+
+def run_gmm_system():
```

Comment:
Is the FSA bug correction included?

Reply:
I am not sure. Where / how would that be included? RASR version?

Reply:
no.

Reply:
now it should be